### ISSUE TITLE
TypeError: bowser_1.default.parse is not a function  

### ISSUE THREAD

**Issue by mattvaughan** [DevExpress/testcafe#4499]
### Are you requesting a feature or reporting a bug?
Bug

### What is the Current behavior?
Running `testcafe chromium listTests.js` throws an error before browser launches:

```
TypeError: bowser_1.default.parse is not a function
node_modules/testcafe/src/utils/parse-user-agent.ts:6:40
```

### What is the Expected behavior?
I expect the browser to launch. This is not an issue with commit 81b5c5be703315e91835a5a9372ecab632b72517 or before.

### How would you reproduce the current behavior (if this is a bug)?
Run a test with any test code

```js
fixture('Test')
  .page('https://google.com/')
  .beforeEach(async t => {
  })

test('Do nothing', t => {
  console.log('hi');
});

```
### Your Environment details:
 
* testcafe version:                   83eafed7a2fd97f0c24d87866a9cc97185d6e712
* node.js version:                    `v12.12.0`
* command-line arguments:  `testcafe chromium listTests.js`
* browser name and version: `chromium 77.0.3865.120`
* platform and version:          `Arch Linux 64 bit`

**Comment by mattvaughan** [comment]
This looks like it was fixed as I was writing the ticket 8a8137361b7739d6458315f0c7faef9ea3ba34d1.

Feel free to close if you aren't experiencing this bug.

**Comment by Farfurix** [comment]
@mattvaughan 

Hello,

Thank you for testing our latest updates. I checked the latest `testcafe@v1.7.0-rc.1` version and there were no errors. My environment details: Ubuntu / Chromium / TestCafe v1.7.0-rc.1.

Result:
![Capture](https://user-images.githubusercontent.com/30019338/69314043-341f5700-0c44-11ea-8c23-a7cf581901bc.PNG)
